game/scripts/npc/npc_items_custom.txt

```diff
@@ -6422,13 +6422,21 @@
 			{
 				"Passive"			"0"
 				"IsHidden"			"0"
-
+				"IsPurgable"		"0"
 				"Properties"
 				{
 					"MODIFIER_PROPERTY_ATTACKSPEED_BONUS_CONSTANT"		"%aura_bonus_attack_speed"
 					"MODIFIER_PROPERTY_MOVESPEED_BONUS_PERCENTAGE"		"%aura_bonus_movement_speed_pct"
 					"MODIFIER_PROPERTY_HEALTH_REGEN_CONSTANT"			"%aura_health_regen"
 				}
+				"OnDestroy"
```

Comment:
What aura effect has to do with unit?
You may listen to `OnDestroy` on `modifier_item_summoned_unit` modifier and kill summon with:
```lua
function KillSummon(keys)
	local caster = keys.summon
	local ability = keys.ability
	local unit = caster["custom_summoned_unit_ability_" .. ability:GetAbilityName()]
	if not IsValidEntity(unit) or not unit:IsAlive() then return end

	unit:ForceKill(false)
end
```